### PR TITLE
add support for centos/rhel 7

### PR DESCRIPTION
**Pull request by jordane**

### CODE REVIEW

File: .kitchen.cloud.yml [hunk 1]
```diff
@@ -13,17 +13,8 @@ driver_config:
...
-      sudo:
-        users:
-          - vagrant
-          - centos
-          - ubuntu
-          - fedora
-        passwordless: true
 
-platforms:
+plaftorms:
```
Comment (ramereth) [inline .kitchen.cloud.yml]: Typo

Comment (jordane) [inline .kitchen.cloud.yml]: fixed

**Comment by ramereth** [comment]
:+1:

**Comment by armiller** [comment]
:+1:

**Pull request merged**